src/User/Credential/EmailCredentialTrait.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MsgPhp\User\Credential;
+
+/**
+ * @author Roland Franssen <franssen.roland@gmail.com>
+ */
+trait EmailCredentialTrait
+{
+    private $email;
+
+    public function getEmail(): string
+    {
+        return $this->email;
```

Comment:
This trait isn't very useful if you want to have a custom `getEmail` I think. Could make this final and simply use `$this->email` in `getUsername`. The same goes for the other traits.

Reply:
:+1: updated.